### PR TITLE
Docs italian version

### PR DESCRIPTION
**Pull request by AngyDev**
Related issue: #24391

**Description**

Add Italian translation of Lights section

### CODE REVIEW

File: docs/list.json [hunk 1]
```diff
@@ -1284,6 +1284,26 @@
 				"PointLightHelper": "api/it/helpers/PointLightHelper",
 				"SkeletonHelper": "api/it/helpers/SkeletonHelper",
 				"SpotLightHelper": "api/it/helpers/SpotLightHelper"
+			},
+
+      "Luci": {
```
Comment (Mugen87) [inline docs/list.json:1289]: Nit: It seems the formatting is slightly off here.

File: docs/list.json [hunk 2]
```diff
@@ -1284,6 +1284,26 @@
...
+				"SpotLight": "api/it/lights/SpotLight"
+			},
+
+			"Luci / Ombre": {
+				"LightShadow": "api/it/lights/shadows/LightShadow",
+				"PointLightShadow": "api/it/lights/shadows/PointLightShadow",
+				"DirectionalLightShadow": "api/it/lights/shadows/DirectionalLightShadow",
+				"SpotLightShadow": "api/it/lights/shadows/SpotLightShadow"
 			}
     }
```
Comment (Mugen87) [inline docs/list.json:1308]: It seems the `it` section contains some soft tabs in this file. Better to always use hard tabs.

Comment (AngyDev) [reply docs/list.json:1308]: I'll check, I don't know what's happened, maybe the JSON formatter. Thanks for the check.

**Comment by AngyDev** [comment]
I hope the indentation is correct now

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**